bench/wtperf/config.c

```diff
@@ -785,6 +815,13 @@ config_sanity(WTPERF *wtperf)
                   workp->table_index, opts->table_count);
                 return (EINVAL);
             }
+            if (workp->modify != 0 && workp->ops_per_txn == 0) {
+                fprintf(stderr,
+                  "Modify operations must be executed with "
+                  "explicit transaction, specify "
+                  "ops_per_txn.");
```

Comment:
With the new format these lines can probably be joined.

Reply:
corrected.